### PR TITLE
chore: use libp2p@2.0

### PR DESCRIPTION
**Pull request by SgtPooki**
## Title

chore: use libp2p@2.0

## Description

Updates the bootstrapper to use libp2p@2.0.

- **chore: update deps**
- **chore: use libp2p@2.0**

## Change checklist

- [ ] I have performed a self-review of my own code
- [ ] I have made corresponding changes to the documentation if necessary (this includes comments as well)
- [ ] I have added tests that prove my fix is effective or that my feature works

### CODE REVIEW

**Review by SgtPooki — COMMENTED** [review]
self review

File: src/index.ts [hunk 1]
```diff
@@ -149,7 +147,7 @@ async function main (): Promise<void> {
       yamux(),
       mplex()
```
Comment (achingbrain) [inline src/index.ts:148]: Maybe remove mplex?

File: src/index.ts [hunk 2]
```diff
@@ -149,7 +147,7 @@ async function main (): Promise<void> {
...
     ],
-    connectionEncryption: [
+    connectionEncrypters: [
       noise()
```
Comment (achingbrain) [inline src/index.ts:151]: Maybe add tls?

**Review by achingbrain — APPROVED** [review]

File: src/index.ts (hunk 2 above)
Comment (SgtPooki) [reply src/index.ts]: fixed in [`4e48bac` (#139)](https://github.com/libp2p/js-libp2p-amino-dht-bootstrapper/pull/139/commits/4e48bac1779267f17f77c5d13a439f17cf89b297)

File: src/index.ts (hunk 1 above)
Comment (SgtPooki) [reply src/index.ts]: fixed in [`4e48bac` (#139)](https://github.com/libp2p/js-libp2p-amino-dht-bootstrapper/pull/139/commits/4e48bac1779267f17f77c5d13a439f17cf89b297)

**Review by wemeetagain — APPROVED** [review]

**Pull request merged**